### PR TITLE
[doc]proofread doc part 3 and 4

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
Changes:
1. Fix format and lint errors.
2. Update ununified contents.

### CODE REVIEW

File: docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ bash> git clone https://github.com/vesoft-inc/nebula.git
 
 #### 步骤 2 : 安装依赖
 
+下方 `build_dep.sh` 脚本依赖 `Open-Storage-Service (OSS)`。
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md]: 这句不太对。删掉比较好。

File: docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 2]
```diff
@@ -70,7 +72,9 @@ bash> make
 bash> sudo make install
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md]: line 66: 步骤 4A: 构建 Debug 版本

File: docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 3]
```diff
@@ -70,7 +72,9 @@ bash> make
...
 ```
 
-#### 构建 Release 版本，重新生成 Makefile
+或构建 Release 版本。
+
+#### 步骤 4: 构建 Release 版本，重新生成 Makefile
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md]: #### 步骤 4B: 构建 Release 版本，重新生成 Makefile

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md [hunk 4]
```diff
@@ -48,14 +48,14 @@ Nebula 使用 `gflags` 进行运行时配置。
 示例
 
 ```ngql
-UPDATE CONFIGS storage:rocksdb_column_family_options =
+nebula> UPDATE CONFIGS storage:rocksdb_column_family_options =
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md]: 加个 \\

File: docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 5]
```diff
@@ -16,31 +16,31 @@ When building type is **DEBUG**, an SSD disk with space more than **50GB** is su
...
 
 ```bash
 bash> git clone https://github.com/vesoft-inc/nebula.git
 ```
 
-#### Step 2: install dependencies
+#### Step 2: Install Dependencies
 
-The script `build_dep.sh` bellow depends on the cloud `Open-Storage-Service (OSS)`
+The script `build_dep.sh` bellow depends on the cloud `Open-Storage-Service (OSS)`.
```
Comment (whitewum) [inline docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md]: ditto

File: docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 6]
```diff
@@ -16,31 +16,31 @@ When building type is **DEBUG**, an SSD disk with space more than **50GB** is su
...
 
-```bash
- bash> cd nebula && ./build_dep.sh
-```
+  ```bash
+   bash> cd nebula && ./build_dep.sh
+  ```
 
-##### B. If you cannot download `OSS` packages directly
+- If you cannot download `OSS` packages.
```
Comment (whitewum) [inline docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md]: ,

File: docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 7]
```diff
@@ -72,15 +72,17 @@ bash> make
 bash> sudo make install
 ```
 
-#### Build release version, regenerate the makefile
+Or you can build the release version.
```
Comment (whitewum) [inline docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md]: ditto

File: docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 8]
```diff
@@ -97,58 +99,27 @@ bash> sudo make install
...
-#### 容器内编译
-
-```bash
-docker> mkdir _build && cd _build
-docker> cmake ..
-docker> make
-docker> make install
-``` -->
-
 #### 启动 nebula 服务
```
Comment (wilsonyou) [inline docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md]: nebula -> Nebula Graph

File: docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 9]
```diff
@@ -97,58 +99,27 @@ bash> sudo make install
...
 
-经过上述的安装后，便可以在容器内部启动 nebula 的服务，nebula 默认的安装目录为 `/usr/local/nebula`
+经过上述的安装后，便可以启动 nebula 的服务，nebula 默认的安装目录为 `/usr/local/nebula`。
```
Comment (wilsonyou) [inline docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md]: Please change all nebula to Nebula Graph

File: docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 10]
```diff
@@ -4,7 +4,7 @@
 
 Nebula is developed based C++14, so it requires a compiler supporting C++14 features.
 
-### Supported system version
+### Supported System Version
```
Comment (wilsonyou) [inline docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md]: Supported System Version -> Supported Systems

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-balance.md [hunk 11]
```diff
@@ -90,7 +90,7 @@ nebula> SHOW HOSTS
 
 ### Step 3 Data migration
```
Comment (wilsonyou) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-balance.md]: Migration

**Pull request merged**